test/strings/strcat.c

```diff
@@ -0,0 +1,17 @@
+#include <string.h>
+#include "smack.h"
+
+// @expect verified
```

Comment:
strcat and strcat_fail should have the same number or loop unrolls. That holds in general for any _fail variety of a file - their number of unrolls should match.